documentation/sphinx/source/release-notes/release-notes-7131.rst

```diff
@@ -0,0 +1,198 @@
+.. _release-notes:
+
+#############
+Release Notes
+#############
+
+71.3.1
+======
+
+Features
+--------
+- Add HoltLinearSmoother class
+- EKP and KMS Health Check
+- Adding blob granule restarting tests to use tenants, encryption
+- Implement IStorageMetricsService interface
+- Make stepSize configurable for preloadApplyMutationsKeyVersionMap
+- Add networkoption to disable non-TLS connections
+- New ConsistencyScan
+- Add WatchMapSize and OutstandingWatches to TransactionMetrics 
+- Add blob manifest url and mlogs url in status json. 
+- Add tenant_id_prefix to metacluster status
+- Support blob granule restore in fdbrestore 
+- Move lastFlushTs to BlobGranuleBackupConfig 
+- Cache change feeds durably on blob workers
+- Support restoring a cluster with a tenant in the error state
+- EaR RESTSimKMS Vault 
+- EaR: Update KMS URL refresh policy and fix bugs 
+- EaR: Add test case to validate decryption with invalid key
+- Decouple token bucket knobs for different types of throttlers 
+- Adding blob ranges to backup keys 
+- Allow reading from system keyspace even with default tenant
+- Add ExpectStableThroughput simulation test 
+- Simplify GlobalTagThrottler limiting TPS calculation 
+
+
+Performance
```

Comment:
A few of these are improving testing, not FDB performance, should probably be moved out of this section:
* Increase coverage for slow tests in TestHarness2
* Speed up BackupCorrectnessClean
* Speed up DR upgrade tests

Reply:
Agreed, I can add FDB Testing as the section (which doesn't exists in past release notes), however, in principal they are performance improvements to FDB repo if taken as a product.

Reply:
(For this and some of the other comments.) Maintaining consistency with past release notes seems like a good reason to leave some of these things as they are right now.